dev/drivers/scripts/prep/global_ens/jevs_global_ens_naefs_atmos_prep.sh

```diff
@@ -24,7 +24,7 @@ export MODELNAME=naefs
 source $HOMEevs/versions/run.ver
 module reset
 module load prod_envir/${prod_envir_ver}
-source $HOMEevs/modulefiles/${COMPONENT}/${COMPONENT}_${STEP}.sh
+source $HOMEevs/dev/modulefiles/${COMPONENT}/${COMPONENT}_${STEP}.sh
 
 export evs_ver=v1.0.0
```

Comment:
@ShelleyMelchior-NOAA Please remove export evs_ver=v1.0.0 in [dev/drivers/scripts/prep/global_ens/jevs_global_ens_naefs_atmos_prep.sh](https://github.com/NOAA-EMC/EVS/pull/354/files#diff-56df572888001debd41ed0a7e26b47fa52512b0c4ddbc2155d0f66525d0e143d). 

CC @PerryShafran-NOAA